logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -551,6 +558,41 @@ internal class MLSConversationDataSource(
             }
         }
 
+    override suspend fun getUserIdentity(userId: UserId) =
+        wrapStorageRequest { conversationDAO.getMLSGroupIdByUserId(userId.toDao()) }.flatMap { mlsGroupId ->
+            mlsClientProvider.getMLSClient().flatMap { mlsClient ->
+                wrapMLSRequest {
+                    mlsClient.getUserIdentities(
+                        mlsGroupId,
+                        listOf(userId.toCrypto())
+                    )[userId.value]!!
+                }
+            }
+        }
+
+    override suspend fun getMembersIdentities(
```

Comment:
question: from the place your passing the conversationId [from the AR] you can make sure that you only pass the groupId, and if the protocol is not MLS you can't call it with a normal conversation Id, wdyt?